infra/blueprint-test/examples/simple_tf_module/main.tf

```diff
@@ -22,6 +22,10 @@ provider "null" {
   version = "~> 2.1"
 }
 
+terraform {
+  backend "local" {}
+}
```

Comment:
What happens if a backend is not defined?

Reply:
Looks like it just gives a warning if we pass in a partial config to local backend.